File: test_project/.vscode/SnipHub/snip/HelloWorld.java
// Java Hello World
public class HelloWorld {
    public static void main(String[] args) {
        System.out.println("Hello, World!");
        
        // 使用方法
        System.out.println(sayHello("Java"));
        
        // 使用对象
        Greeter greeter = new Greeter("Java OOP");
        System.out.println(greeter.greet());
    }
    
    public static String sayHello(String name) {
        return "Hello, " + name + "!";
    }
}

// Greeter类
class Greeter {
    private String name;
    
    public Greeter(String name) {
        this.name = name;
    }
    
    public String greet() {
        return "Hello, " + name + "!";
    }
}

File: test_project/.vscode/SnipHub/snip/HelloWorld.cs
// C# Hello World
using System;

namespace HelloWorldApp
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello, World!");
            
            // 使用方法
            Console.WriteLine(SayHello("C#"));
            
            // 使用类
            var greeter = new Greeter("C# OOP");
            Console.WriteLine(greeter.Greet());
        }
        
        static string SayHello(string name)
        {
            return $"Hello, {name}!";
        }
    }
    
    class Greeter
    {
        private string _name;
        
        public Greeter(string name)
        {
            _name = name;
        }
        
        public string Greet()
        {
            return $"Hello, {_name}!";
        }
    }
}
